types/shelljs/shelljs-tests.ts

```diff
@@ -89,24 +89,24 @@ const testPath = shell.env["path"];
 const version = shell.exec("node --version").stdout;
 
 // $ExpectType ShellString
-const version2 = shell.exec("node --version", {async: false});
+const version2 = shell.exec("node --version", { async: false });
```

Comment:
I&nbsp;would prefer if&nbsp;you didn’t reformat the&nbsp;tests in&nbsp;this&nbsp;PR as&nbsp;it&nbsp;adds too&nbsp;much unnecessary noise.

Reply:
Sorry about that. I guess now I'll leave it like this or do you want me to revert those changes? I could commit these changes separately from the others you suggest so to minimise the overall noise and to get it back the way it once was. Let me know.

Reply:
Just went ahead and reverted the formatting. Noise cancellation: on/~~off~~

Reply:
Wow! Given #34364, I was sure you would gonna love the whitespace changes, @ExE-Boss :stuck_out_tongue:

Reply:
Well, the difference between this and #34364 is that in #34364, it was gonna be a ~90% diff either way, so I decided to just convert it to tabs.

This is a PR that affects a single thing, so reformatting the tests is more or less out of scope.